addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/Base64Disclosure.java

```diff
@@ -101,8 +105,8 @@ public String getName() {
     public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 
         log.debug("Checking message {} for Base64 encoded data", msg);
-
-        // get the body contents as a String, so we can match against it
+        IGNORED_HEADERS.forEach(header -> msg.getResponseHeader().setHeader(header, null));
```

Comment:
The original message should not be changed, use/edit a copy of the header instead.